vc17/theforgottenserver.vcxproj

```diff
@@ -353,4 +354,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Pls keep trailing newline

Reply:
I think it is automated by visual studio to format it like that, weird I know

Reply:
Yes, this is automatic. I can even fix it, but it will come back on the next change and will have to be fixed again.

Reply:
VS is stupid